src/event-log/event-log-level.ts

```diff
@@ -60,6 +64,17 @@ export class EventLogLevel implements EventLog {
     await watermarkLog.put(watermark, messageCid);
     await cidLog.put(messageCid, watermark);
 
+    // if (this.config.eventStream) {
+    //   @andorsk: Either add signing or allow for event creation without signing.
+    //   const logMessage = await EventMessage.create({
+    //     descriptor: {
+    //         type: EventType.Log,
+    //         messageTimestamp: "asfd",
+    //     },
+    //     authorizationSignatureInput : Jws.createSignatureInput(tennat),
+    //   })
+    //   this.config.eventStream.add(logMessage)
```

Comment:
Sanity: I don't think we're not able to sign here because `EventLog.append()` is called from within the handler after the message has already been signed and received by the DWN.

But I think you're right, not sure we need to sign this at all.

Reply:
yea...i fixed this by just forwarding the message itself for now.